meta-xt-domx/recipes-extended/xen/xen-source.inc

```diff
@@ -1,9 +1,10 @@
 LIC_FILES_CHKSUM = "file://COPYING;md5=419739e325a50f3d7b4501338e44a4e5"
 
 XEN_URL ??= "git://github.com/xen-troops/xen.git;protocol=https;branch=xen-4.16rc-migration"
+XEN_REV ??= "${AUTOREV}"
 
 SRC_URI = "${XEN_URL}"
 XEN_REL = "4.16"
 PV = "${XEN_REL}.0+git${SRCPV}"
-SRCREV = "${AUTOREV}"
+SRCREV = "${XEN_REV}"
```

Comment:
Adding separate variable allow me to pass this variable from moulin yaml

Reply:
Are you going to pass all components revisions from YAML file?

Xen, u-boot, linux, ARM TF, optee, libc, libalsa, etc? Or why Xen is so special?

Reply:
Because XEN has XEN_URL and XEN_REL, see bff53e66 for the details. BTW linux also have KERNEL_SRC and SRC_BRANCH special variables (linux-imx_5.10.bb recipe). I think this is also an issue. Why make SRC_BRANCH configurable and hardcode SRCREV? So from my standpoint if XEN have global URL, then why shouldn't revision be global as well? This will give us more room for configuration.

Reply:
Well, that was a bad idea in the first place.
But, as it is already present, I'm fine with it.